### PR TITLE
SDIT-928 Basic hearing mapping

### PR DESCRIPTION
**Pull request by andymarke**
TODO

- Mapping when type is NULL
- Results

### CODE REVIEW

File: src/test/kotlin/uk/gov/justice/digital/hmpps/prisonerfromnomismigration/adjudications/AdjudicationTransformationTest.kt [hunk 1]
```diff
@@ -589,6 +593,77 @@ class AdjudicationTransformationTest {
...
+
+      assertThat(dpsAdjudication.hearings).containsExactly(
+        MigrateHearing(
+          oicHearingId = 54321,
+          oicHearingType = GOV, // TODO - we always havce a NOMIS type so default to this until we have a decision
+          hearingDateTime = "2021-01-01T12:00:00",
+          adjudicator = null,
+          commentText = null,
+          locationId = 321,
+          hearingResult = null,
```
Comment (mikehalmamoj) [inline src/test/kotlin/uk/gov/justice/digital/hmpps/prisonerfromnomismigration/adjudications/AdjudicationTransformationTest.kt:662]: Do you need a test for when this is not null?

**Review by mikehalmamoj — APPROVED** [review]

File: src/test/kotlin/uk/gov/justice/digital/hmpps/prisonerfromnomismigration/adjudications/AdjudicationTransformationTest.kt (hunk 1 above)
Comment (andymarke) [reply src/test/kotlin/uk/gov/justice/digital/hmpps/prisonerfromnomismigration/adjudications/AdjudicationTransformationTest.kt:662]: 100% that is the next phase - there is a whole thing about results not touched yet
